contrib/generate_profile.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+fdbdir=`pwd`/..
+export LD_LIBRARY_PATH=$fdbdir/lib:$LD_LIBRARY_PATH
+export FDB_CLUSTER_FILE=$fdbdir/fdb.cluster
+export LLVM_PROFILE_FILE=$fdbdir/sandbox/fdb-%p.profraw
+../bin/fdbmonitor --conffile ../sandbox/foundationdb.conf --lockfile ../sandbox/fdbmonitor.pid &
```

Comment:
Similar to the comment above, let's not use `..`, but some CMake variable instead

Reply:
done